app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -26,13 +26,13 @@ def build_for_importer
 
     def parent_jobs
       self.parsed_metadata[related_parents_parsed_mapping].each do |parent_identifier|
-        CreateRelationshipsJob.perform_later(entry_identifier: self.identifier, parent_identifier: parent_identifier, importer_run: self.last_run)
+        CreateRelationshipsJob.perform_later(entry_identifier: self.identifier, parent_identifier: parent_identifier, importer_run: self.last_run) if parent_identifier.present?
```

Comment:
Would you mind also including [this change](https://github.com/samvera-labs/bulkrax/pull/388/files#diff-b28d7bc5e71422d28ccdcbda7f6f294700f0f12553cc2820e0cf02cca25cb35cR14-L16) in this PR? I should have put it in a smaller bug fix PR like this and it's related